doc/shared/FAQ.rst

```diff
@@ -0,0 +1,335 @@
+..
+   ----------------------------------------------------------------
+   SUNDIALS Copyright Start
+   Copyright (c) 2002-2024, Lawrence Livermore National Security
+   and Southern Methodist University.
+   All rights reserved.
+
+   See the top-level LICENSE and NOTICE files for details.
+
+   SPDX-License-Identifier: BSD-3-Clause
+   SUNDIALS Copyright End
+   ----------------------------------------------------------------
+
+.. _FAQ:
+
+##########################
+Frequently Asked Questions
+##########################
+
+The following are some questions frequently asked by SUNDIALS users.
+If you do not see your question here, please do not hesistate to ask the
+SUNDIALS mailing list by emailing your question to ``sundials-users@llnl.gov``,
+or by opening an issue on our GitHub at `https://github.com/LLNL/sundials`.
+
+Installation Related
+--------------------
+
+.. collapse:: I have the sundials source, how do I build it?
+
+  See the :ref:`Installation.CMake` section.
```

Comment:
I think indentation alignment only matters in the latex builds, but the `See` should be three spaces in so it aligns with `collapse` (and similarly below).